specification/containerservice/resource-manager/Microsoft.ContainerService/aks/readme.md

```diff
@@ -37,15 +37,25 @@ openapi-type: arm
 tag: package-2024-01
 ```
 
-
 ### Tag: package-2024-01
 
 These settings apply only when `--tag=package-2024-01` is specified on the command line.
 
 ```yaml $(tag) == 'package-2024-01'
 input-file:
   - stable/2024-01-01/managedClusters.json
+tag: package-preview-2024-01
```

Comment:
This line of `tag` should be removed to resolve the SDK generation failures.

Reply:
Thanks for catching that!